src/sas/example_data/media/testdata_help.rst

```diff
@@ -297,39 +358,6 @@ spheres2micron_long
 
 .. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
 
-DLS Data
-^^^^^^^^
-DLS (Dynamic Light Scattering) data sets primarily contain the intensity
-autocorrelation function as a function of the delay time (in microseconds).
-
-DLS Test Data
-................
-**Example data files are located in the \\dls_data sub-folder.**
-
-dls_monodisperse / dls_polydisperse
-  - DLS data from a very dilute dispersion of 3 nm polymer latex nanoparticles in
-    h/d-water.
-  - Suitable for testing the cumulants_dls model.
-
-.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
-
-NR Data
-^^^^^^^
-Neutron (NR) or X-ray (XR) Reflectometry data sets primarily contain the
-interfacial reflectivity as a function of *Q*.
-
-.. note:: The Refl1D reflectivity model-fitting software uses the same fitting
-         engine (Bumps) as SasView.
-
-NR Test Data
-................
-**Example data files are located in the \\nr_data sub-folder.**
-
-NR_Ni_down_state / NR_Ni_up_state
-  - Polarised (spin down/up) NR data from a Ni multilayer sample.
-
-.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
-
 Save States
```

Comment:
Should be saved states, as in the description below?

Reply:
Changed.

Folder named also changed.